crates/turbopack-ecmascript/tests/tree-shaker/analyzer/complex/output.dot

```diff
@@ -0,0 +1,19 @@
+digraph {
```

Comment:
[Visualized](https://dreampuf.github.io/GraphvizOnline/#digraph%20%7B%0A%20%20%20%200%20%5B%20label%20%3D%20%22%5C%22getDog%231%5C%22%22%20%5D%0A%20%20%20%201%20%5B%20label%20%3D%20%22%5C%22dog%231%5C%22%22%20%5D%0A%20%20%20%202%20%5B%20label%20%3D%20%22%5C%22setDog%231%5C%22%22%20%5D%0A%20%20%20%203%20%5B%20label%20%3D%20%22%5C%22newDog%232%5C%22%22%20%5D%0A%20%20%20%204%20%5B%20label%20%3D%20%22%5C%22dogRef%231%5C%22%22%20%5D%0A%20%20%20%205%20%5B%20label%20%3D%20%22%5C%22cat%231%5C%22%22%20%5D%0A%20%20%20%206%20%5B%20label%20%3D%20%22%5C%22initialCat%231%5C%22%22%20%5D%0A%20%20%20%207%20%5B%20label%20%3D%20%22%5C%22getChimera%231%5C%22%22%20%5D%0A%20%20%20%200%20-%3E%201%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%201%2C%20initial%3A%200%20%7D%22%20%5D%0A%20%20%20%202%20-%3E%203%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%202%2C%20initial%3A%200%20%7D%22%20%5D%0A%20%20%20%202%20-%3E%201%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%201%2C%20initial%3A%200%20%7D%22%20%5D%0A%20%20%20%204%20-%3E%201%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%200%2C%20initial%3A%201%20%7D%22%20%5D%0A%20%20%20%204%20-%3E%200%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%200%2C%20initial%3A%201%20%7D%22%20%5D%0A%20%20%20%204%20-%3E%202%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%200%2C%20initial%3A%201%20%7D%22%20%5D%0A%20%20%20%206%20-%3E%205%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%200%2C%20initial%3A%201%20%7D%22%20%5D%0A%20%20%20%207%20-%3E%205%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%201%2C%20initial%3A%200%20%7D%22%20%5D%0A%20%20%20%207%20-%3E%201%20%5B%20label%20%3D%20%22VarInfo%20%7B%20live%3A%201%2C%20initial%3A%200%20%7D%22%20%5D%0A%7D%0A)

Reply:
Since github can visualize graphs it might be cool to emit this as Markdown with the graph embedded.

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams

Reply:
Do you mean printing it from the GitHub action?

Reply:
Ah, I understood. I'll do it.